src/lib/boilerplate-install.ts

```diff
@@ -64,13 +54,13 @@ async function installBoilerplate (context) {
     print.error('Error call stack:')
     print.error(e.stack)
     spinner.fail(`error loading the boilerplate`)
-    process.exit(exitCodes.PLUGIN_INVALID)
+    return false
   }
 
   // must have an `install` function
   if (!pluginModule.hasOwnProperty('install')) {
     spinner.fail(`boilerplate.js is missing a 'install' function`)
-    process.exit(exitCodes.PLUGIN_INVALID)
+    return false
```

Comment:
Are we losing exit code info by doing this?

Reply:
So I forgot to mention that this file _used_ to be called as a spawned process command. I switched it to a simple async function call to reduce overhead and because it apparently now works fine. Due to this, we no longer should kill the process.

The exit code wasn't particularly helpful anyway. More important is showing the error message.

Good question though.